pkg/core/tekton/tekton.go

```diff
@@ -292,8 +326,10 @@ func generatePipelineRun(p *v1beta1.Pipeline, codeset domain.Codeset) (*v1beta1.
 			switch param.Name {
 			case codesetNameParam:
 				prb.Param(param.Name, codeset.Name)
-			case codesetVersion:
+			case codesetVersionParam:
 				prb.Param(param.Name, codesetVersion)
+			case codesetProjectParam:
+				prb.Param(param.Name, codeset.Project)
```

Comment:
I think I know the answer, but just to confirm: you're adding this parameter to the pipeline run only because you're using it to be able to query pipeline runs by codeset project, right ? and you're using the parameter in the expression to which you're setting the label's value.

Reply:
yes